en/development/testing.rst

```diff
@@ -501,6 +501,10 @@ table linked to the model called Article. You can use any model available in
 your application. The statement will only import the Article schema, and  does
 not import records. To import records you can do the following::
 
+.. versionchanged:: 2.7 
+    ``model`` has been replaced by ``table``
```

Comment:
I just doubled checked again... and it is working today...

I'm sorry if i wasted your time....

I believe my issue might have been that I am building a Server / Client
system and this forces me to start up processes in the background...
without the original testing framework.... So I have to pass along a
variable to start up the test system ..... I'm noticing when components are
not fully loaded from the test I get background failures , which could have
interfered with the 'model' import but not the 'table' import.... Today I
stripped my system down so a very clean set of functions and I'm monitoring
it.

Once again sorry about that...



---
Sacha Lewis Dmytruk - President
*undoLogic inc. *
Montreal: 1.514.488.5775 ext 6
Toll-Free: 1.855.488.5775 ext 6
undoLogic.com <http://undologic.com/>

On Sat, Aug 29, 2015 at 5:03 PM, Mark Story <notifications@github.com>
wrote:

> In en/development/testing.rst
> <https://github.com/cakephp/docs/pull/3192#discussion_r38265930>:
>
> > @@ -501,6 +501,10 @@ table linked to the model called Article. You can use any model available in
> >  your application. The statement will only import the Article schema, and  does
> >  not import records. To import records you can do the following::
> >
> > +.. versionchanged:: 2.7
> > +    ``model`` has been replaced by ``table``
>
> I'll take a look. Import with model should still work as there hasn't been
> any intentional changes to remove that functionality.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/docs/pull/3192/files#r38265930>.
>